lib/src/ast/expression.dart

```diff
@@ -2,21 +2,41 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// TODO: This is a placeholder.
-//
-// Probably want to either:
-// - Port the original one
-// - Write a new one
-// - Use the Dart analyzer and special case pipes
-// - Use the Dart analyzer and transform pipes
+import 'package:analyzer/analyzer.dart';
+import 'package:angular_ast/src/expression/parser.dart';
+
+/// Wraps a parsed Dart [Expression] as an Angular [ExpressionAst].
 class ExpressionAst {
-  final String expression;
+  /// Dart expression.
+  final Expression expression;
```

Comment:
just from a readability standpoint, importing this as dart and using `dart.Expression` may be a hair clearer